wordle-web/pages/index.vue

```diff
@@ -1,95 +1,15 @@
 <template>
-  <v-container>
-    <v-progress-linear v-if="game.isBusy" color="primary" indeterminate />
-    <v-card v-else class="text-center">
-      <v-alert
-        v-if="game.gameState != GameState.Playing"
-        :color="game.gameState == GameState.Won ? 'success' : 'error'"
-        class="mb-5"
-        tile
-      >
-        <h3>
-          You've
-          {{ game.gameState == GameState.Won ? "Won! 🥳" : "Lost... 😭" }}
-        </h3>
-        <v-card-text>
-          The word was: <strong>{{ game.secretWord }}</strong>
+    <v-card class="text-center">
```

Comment:
WIll you be implementing the choice to start word of the day vs never ending mode on your start page? Or is that only available in the menu?

Reply:
There are two game pages one with a random word and one that is the word of the day

Reply:
@MattA41 yes, but we should probably put some links to each of them on our landing page instead of burying them both in the hamburger menu.